pootle_pytest/fixtures/site.py

```diff
@@ -182,3 +183,77 @@ def site_matrix_with_announcements(site_matrix):
                 'guide</a>.</div>'),
             virtual_path="announcements/%s/%s"
             % (tp.language.code, tp.project.code))
+
+
+def _update_submission_times(update_time, last_update=None):
+    from pootle_statistics.models import Submission
+
+    submissions = Submission.objects.all()
+    if last_update:
+        submissions = submissions.exclude(
+            creation_time__lte=last_update)
+    submissions.update(creation_time=update_time)
+
+
+def _add_submissions(unit, created):
+    from pootle_store.models import UNTRANSLATED, FUZZY, OBSOLETE, Unit
+
+    from django.contrib.auth import get_user_model
+
+    original_state = unit.state
+    unit.created = created
+
+    User = get_user_model()
+    admin = User.objects.get(username="admin")
+    member = User.objects.get(username="member")
+    member2 = User.objects.get(username="member2")
+
+    first_modified = created + timedelta(days=((30 * unit.index) + 10))
+
+    # add suggestion at first_modified
+    suggestion, _ = unit.add_suggestion(
+        "Suggestion for %s" % unit.source,
+        user=member,
+        touch=False)
+    _update_submission_times(first_modified, created)
+
+    # accept the suggestion 7 days later if not untranslated
+    next_time = first_modified + timedelta(days=7)
+    if original_state == UNTRANSLATED:
+        unit.reject_suggestion(
+            suggestion, unit.store.translation_project, admin)
+    else:
+        unit.accept_suggestion(
+            suggestion, unit.store.translation_project, admin)
+        Unit.objects.filter(pk=unit.pk).update(
+            submitted_on=next_time, mtime=next_time)
+    _update_submission_times(
+        next_time, first_modified)
+
+    # add another suggestion as different user 7 days later
+    suggestion2, _ = unit.add_suggestion(
+        "Suggestion 2 for %s" % unit.source,
+        user=member2,
+        touch=False)
+    _update_submission_times(
+        first_modified + timedelta(days=14),
+        next_time)
+
+    # mark FUZZY
+    if original_state == FUZZY:
+        unit.markfuzzy()
+
+    # mark OBSOLETE
+    if original_state == OBSOLETE:
+        unit.makeobsolete()
+
+
+@pytest.fixture
+def site_matrix_with_submissions(site_matrix):
+    from pootle_store.models import Unit
+
+    year_ago = datetime.now() - timedelta(days=365)
```

Comment:
Even if there might not be noticeable effects in tests, I'd rather use `django.utils.timezone.now()` everywhere to better align with the rest of the app code.